docs/_future/request_time_off_tutorial.md

```diff
@@ -321,9 +321,9 @@ Once the function is defined, we need to create a listener for it to perform any
 
 The function listener is declared in the file [`listeners/functions/request-approval.js`](https://github.com/slack-samples/bolt-js-request-time-off/blob/main/listeners/functions/request-approval.js).
 
-In the `request-approval.js` file, we are defining a handler function, `notifyApprover`, that our listener will run when the `ApprovalFunction` is invoked. `notifyApprover` takes in the provided parameters from the function and calls the API method [`client.chat.postMessage`](https://api.slack.com/methods/chat.postMessage) to send a [Block Kit](https://api.slack.com/block-kit) message with the submitted time off information to the manager. Additional buttons to approve or deny the request are also added in to the message Block Kit&mdash;these will be handled by [Action](#action) handlers that are covered in the next section.
+In the `request-approval.js` file, we are defining a handler function, `notifyApprover`, that our listener will run when the `ApprovalFunction` is invoked. `notifyApprover` takes in the provided parameters from the function and calls the API method [`client.chat.postMessage`](https://api.slack.com/methods/chat.postMessage) to send a [Block Kit](https://api.slack.com/block-kit) message with the submitted time off information to the manager. Additional buttons to approve or deny the request are also added into the message Block Kit&mdash;these will be handled by [Action](#action) handlers that are covered in the next section.
 
-This `notifyApprover` handler function is then passed in to a new `SlackFunction` registration so that it will be called when the `ApprovalFunction is invoked:
+This `notifyApprover` handler function is then passed into a new `SlackFunction` registration so that it will be called when the `ApprovalFunction is invoked:
```

Comment:
```suggestion
This `notifyApprover` handler function is then passed in to a new `SlackFunction` registration so that it will be called when the `ApprovalFunction is invoked:
```